src/MySql/DataLayer.php

```diff
@@ -253,9 +253,11 @@ public static function executeSingleton1($query)
   public static function getTableColumns($schemaName, $tableName)
   {
     $sql = sprintf('
-select COLUMN_NAME as column_name
-,      COLUMN_TYPE as column_type
-,      IS_NULLABLE as is_nullable
+select COLUMN_NAME        as column_name
+,      COLUMN_TYPE        as column_type
+,      IS_NULLABLE        as is_nullable
+,      CHARACTER_SET_NAME as char_set
+,      COLLATION_NAME     as collation
```

Comment:
Don't introduce new synonyms for columns. Use:
* character_set_name
* collation_name